README.md

```diff
@@ -518,6 +518,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Cep.la](http://cep.la/) | Brazil RESTful API to find information about streets, zip codes, neighborhoods, cities and states | No | No | Unknown |
 | [CitySDK](http://www.citysdk.eu/citysdk-toolkit/) | Open APIs for select European cities | No | Yes | Unknown |
 | [Country](http://country.is/) | Get your visitors' country from their IP | No | Yes | Yes |
+| [CountryStateCity](https://countrystatecity.in/) | World countries, states, regions, provinces, cities, towns in JSON, SQL, XML, PLIST, YAML, and CSV formats. | `apiKey` | Yes | Yes |
```

Comment:
- description should not end with `.`
- description should not exceed 100 characters (currently 107)

```suggestion
| [CountryStateCity](https://countrystatecity.in/) | World countries, states, regions, provinces, cities, towns in many formats | `apiKey` | Yes | Yes |
```